interfaces/cython/cantera/composite.py

```diff
@@ -108,6 +108,19 @@ class Solution(Transport, Kinetics, ThermoPhase):
     __slots__ = ()
 
 
+class Plasma(Transport, Kinetics, PlasmaPhase):
+    """
+    A class for plasma-reacting solutions. Class `Plasma` derives from classes `PlasmaPhase`, `Kinetics`, and
+    `Transport`.
+
+    The most common way to instantiate `Plasma` objects is by using a phase
+    definition, species and reactions defined in an input file::
+
+        gas = ct.Plasma('oxygen-plasma.yaml')
+    """
+    __slots__ = ()
```

Comment:
A similar point can be made here. `Plasma` is just a specialization of a `ThermoPhase`, right?

Reply:
How do I do this? Do I just add the functions in PlasmaPhase to ThermoPhase as miscellaneous properties?

Reply:
Looking at this some more, doing this may be indeed more difficult than I thought initially. I think leaving this separate is :+1:, as similar things are done in other instances (on the `Kinetics` side, there is a separate `InterfaceKinetics`).